apps/src/lib/kits/maker/boards/microBit/MicroBitBoard.js

```diff
@@ -31,7 +35,9 @@ export default class MicroBitBoard extends EventEmitter {
 
     this.chromeOS = isChromeOS();
 
-    let portType = serialPortType(true);
+    const portType = isWebSerialPort(port)
+      ? navigator.serial
+      : serialPortType();
```

Comment:
If `navigator.serial` is defined, it is a `Serial` object which represents the entry point into the [Web Serial API](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/serial).